.github/workflows/ci.yml

```diff
@@ -44,14 +44,15 @@ jobs:
 
   test:
     name: Test
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     needs: lint
     permissions:
       contents: read
       checks: write
       pull-requests: write
     strategy:
       matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
we also have mac tests, this should probably be included here as well? can be a follow on.